bower.json

```diff
@@ -21,6 +21,8 @@
     "firebase-element": "PolymerElements/firebase-element#^1.0.12"
   },
   "resolutions": {
+    "iron-location": "1 - 2",
```

Comment:
That's what bower made for me :)

Also https://semver.npmjs.com/ is amazing for playing with semver expressions.